src/app/onboarding/components/glam/archive-type-select-dialog/archive-type-select-dialog.component.scss

```
@@ -5,8 +5,8 @@
   background: #fff;
   border-radius: 12px 12px 0px 0px;
```

```suggestion
  border-radius: 12px;
```
The design for this dialog actually changed at some point after it was initially implemented. It is now not flush with the bottom of the screen, so I believe we have a consistent border radius now.